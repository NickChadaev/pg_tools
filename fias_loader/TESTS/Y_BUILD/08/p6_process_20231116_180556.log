2023-11-16 18:05:56 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_08 postgres /home/n.chadaev@abrr.local/Y_BUILD/08/stage_gar_c_08.csv 2023-11-13
--------------------------------------------------------
[18:05:56] ... Калмыкия Респ, id_region: 9, Execute: /home/n.chadaev@abrr.local/Y_BUILD/08/stage_gar_c_08.csv, process_id:25345
[18:05:56] ... Start process
[18:05:56] ...  -- Version
[18:05:56] ...  -- Очистка от данных
[18:05:56] ...  -- Убираем индексы
[18:05:56] ...  -- Set UNLOGGED
[18:05:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[18:05:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[18:05:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[18:05:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[18:05:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[18:05:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[18:05:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[18:05:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[18:05:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[18:05:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[18:05:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_REESTR_OBJECTS_20231113_a4d42606-daec-4ca7-8342-7e44028a1e37.XML
[18:07:30] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_ADDR_OBJ_20231113_a7d87793-2756-401b-b1cc-540ad872d57f.XML
[18:07:34] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_HOUSES_20231113_29481740-4af6-4858-8e2b-f7ffac31548f.XML
[18:09:32] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_STEADS_20231113_a9013493-6c9e-47fc-afa2-e486353a7924.XML
[18:09:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_APARTMENTS_20231113_b8b7cbef-4d2a-4247-aab2-56748ca0bd01.XML
[18:10:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_ROOMS_20231113_a3b2de00-7d3e-488b-9ba3-29493ec9bafe.XML
[18:10:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_CARPLACES_20231113_78339286-46ef-43b5-a6e8-19d64aaea83f.XML
[18:10:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_ADM_HIERARCHY_20231113_496b0a97-a2ae-4294-a27a-4ede23cda74d.XML
[18:12:12] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_MUN_HIERARCHY_20231113_c59b5548-d0c3-4c88-a524-918b6e77d064.XML
[18:14:02] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_CHANGE_HISTORY_20231113_ef1b249f-2942-435d-9ac7-60f2a3128199.XML
[18:20:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_ADDR_OBJ_DIVISION_20231113_efb4f06a-6c66-4f3c-bb4c-937dad559171.XML
[18:20:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_ADDR_OBJ_PARAMS_20231113_7ad1f0fc-13b9-498c-92cc-b2f4fe93acfb.XML
[18:21:07] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_STEADS_PARAMS_20231113_e4801f55-61dc-457b-9f2c-80ef822f7760.XML
[18:21:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_HOUSES_PARAMS_20231113_447f3b46-80ac-426a-8cc2-60314f9e4e3b.XML
[18:33:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_APARTMENTS_PARAMS_20231113_ef32b47a-da3f-4f8d-b56b-2854f412e56c.XML
[18:34:35] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_ROOMS_PARAMS_20231113_d2e65fce-57db-4134-88a3-dd87e0797d1a.XML
[18:34:35] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_CARPLACES_PARAMS_20231113_4d7f4960-b2ad-471a-b33b-fca78f6d4830.XML
[18:34:35] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/08/AS_NORMATIVE_DOCS_20231113_078d99e3-8aa7-478a-9f94-2fe2cd4bbdc6.XML
[18:35:07] ...  -- Data finish
[18:35:07] ...  -- Set LOGGED
[18:35:11] ... Stop process
--------------------------------------------------------
