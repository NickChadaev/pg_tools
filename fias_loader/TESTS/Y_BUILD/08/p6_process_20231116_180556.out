 save_gar_version 
------------------
               85
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                85
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 135182
gar_fias.as_addr_obj: 5012
gar_fias.as_houses: 146573
gar_fias.as_steads: 986
gar_fias.as_apartments: 54751
gar_fias.as_rooms: 60
gar_fias.as_carplaces: 0
gar_fias.as_adm_hierarchy: 135946
gar_fias.as_mun_hierarchy: 135850
gar_fias.as_change_history: 492003
gar_fias.as_addr_obj_division: 8
gar_fias.as_addr_obj_params: 57840
gar_fias.as_steads_params: 10070
gar_fias.as_houses_params: 985583
gar_fias.as_apartments_params: 90260
gar_fias.as_rooms_params: 64
gar_fias.as_carplaces_params: 0
gar_fias.as_normative_docs: 41307
